tests/functional/event_handler/test_openapi_validation_middleware.py

```diff
@@ -1040,3 +1069,24 @@ def my_path(
 
     result = app(event, {})
     assert result["statusCode"] == 200
+
+
+def test_validation_with_http_single_param():
+    # GIVEN a HTTP API V2 proxy type event
+    app = APIGatewayHttpResolver(enable_validation=True)
+    event = deepcopy(LOAD_GW_EVENT_HTTP)
```

Comment:
can we perhaps use a plain object to make it more explicit we are testing a single queryString with multiple values?

the `LOAD_GW_EVENT_HTTP` or `load_event(...)` will hide that explicitness. When that happens, the test name will need adjustment too.

thank you <3

Reply:
I've moved everything to pytest fixtures and put them on `conftest.py` so they can be reused on all the event handler tests.